app/scripts/untracked_files

```diff
@@ -45,7 +45,7 @@ $ignored_files = [
     'privacy/notices/firefox-hello.lang',
     'privacy/notices/firefox.lang',
 ];
-$ignored_locales = ['ja'];
+$ignored_locales = ['hi', 'ja'];
```

Comment:
Why are we ignoring hi? There's probably a reason, but it's not obvious to me :)